grc/templates/upload/_upload-files-section.html

```diff
@@ -85,6 +98,14 @@ <h2 class="govuk-heading-m">
                            accept=".jpg,.jpeg,.bmp,.png,.tif,.tiff,.pdf"
                            class="govuk-file-upload govuk-!-margin-bottom-4 {{ 'govuk-input--error' if form.documents.errors }}"
                            onchange="checkFile(event)">
+
+                    <div id="enter_password" style="display: none; margin: 10px 0 40px 0;">
+                        <label for="document_password" class="govuk-label govuk-date-input__label">
+                            A password may be required for us to open your file. If this is the case, please enter the password below (Optional)
+                        </label>
+                        <input id="document_password" type="password"
+                            name="document_password" class="govuk-input govuk-date-input__input">
+                    </div>

```

Comment:
This code is `display: none;`.

I'm happy for you to keep it on a branch (in case we want it later), but if we merge it into the main branch, it will be confusing for us and future developers

Reply:
This bit (and the javascript below) don't work (because we don't have the associated python code to remove the password).
So I guess this isn't a section that you regularly test with, so can be removed?